packages/grafana-prometheus/src/querybuilder/parsing.test.ts

```diff
@@ -207,23 +207,25 @@ describe('buildVisualQueryFromString', () => {
       ).toMatchObject({
         errors: [
           {
-            from: 8,
+            from: 4,
             text: 'Query parsing is ambiguous.',
-            to: 68,
+            to: 69,
           },
         ],
       });
     });
 
     it('throws error when visual query parse with aggregation is ambiguous (scalar)', () => {
       expect(buildVisualQueryFromString('topk(5, 1 / 2)')).toMatchObject({
-        errors: [
-          {
-            from: 8,
-            text: 'Query parsing is ambiguous.',
-            to: 13,
-          },
-        ],
+        errors: [],
+        query: {
+          metric: '',
+          labels: [],
+          operations: [
+            { id: '__divide_by', params: [2] },
+            { id: 'topk', params: [5] },
+          ],
+        },
```

Comment:
I think this is wrong. This should return the error. `1 / 2` ir not a valid metric and in the operation list we cannot even show `1`.

Reply:
But on the otherhand this is a valid promql. That'll return nothing but this is valid.